dynamic-layers/qt5-layer/recipes-qt/qt5/qtbase_%.bbappend

```diff
@@ -16,16 +16,33 @@ SRC_URI_append_imxgpu3d = " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'x11', '', '${SRC_URI_APPEND_3D_NOT_X11}', d)} \
 "
 
+PACKAGECONFIG_GBM_IMXGPU_MX8_mx8 = "gbm kms"
+PACKAGECONFIG_GBM_IMXGPU_imxgpu  = "${PACKAGECONFIG_GBM_IMXGPU_MX8}"
+
 PACKAGECONFIG_GL_imxpxp   = "gles2"
-PACKAGECONFIG_GL_imxgpu2d = "${@bb.utils.contains('DISTRO_FEATURES', 'x11', ' gl', '', d)}"
-PACKAGECONFIG_GL_imxgpu3d = "gles2"
+PACKAGECONFIG_GL_imxgpu2d = "${@bb.utils.contains('DISTRO_FEATURES', 'x11', ' gl', '', d)} \
+                             ${PACKAGECONFIG_GBM_IMXGPU}"
+PACKAGECONFIG_GL_imxgpu3d = "gles2 \
+                             ${PACKAGECONFIG_GBM_IMXGPU}"
 PACKAGECONFIG_GL_use-mainline-bsp ?= "gles2 gbm kms"
 
+PACKAGECONFIG_PLATFORM_IMXGPU_MX8_mx8 = "eglfs"
+
 PACKAGECONFIG_PLATFORM          = ""
-PACKAGECONFIG_PLATFORM_imxgpu2d = "no-opengl linuxfb"
+PACKAGECONFIG_PLATFORM_imxgpu2d = " \
+    no-opengl linuxfb \
+    ${PACKAGECONFIG_PLATFORM_IMXGPU_MX8}"
 PACKAGECONFIG_PLATFORM_imxgpu3d = " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'x11',     '', \
        bb.utils.contains('DISTRO_FEATURES', 'wayland', '', \
-                                                       'eglfs', d), d)}"
+                                                       'eglfs', d), d)} \
+    ${PACKAGECONFIG_PLATFORM_IMXGPU_MX8}"
```

Comment:
This is enabled for distro features, why the difference?

Reply:
This can be dropped, I think.

Reply:
Okay, I asked Prabhu and I guess the reason we add it is we use it to test eglfs support. We can keep it locally if you don't want it?